src/controllers/controlpickermenu.cpp

```diff
@@ -358,6 +358,14 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
                        tr("Load Track Into Stopped Deck"),
                        tr("Load selected track into first stopped deck"),
                        m_libraryStr, libraryMenu);
+    addPrefixedControl("[Playlist]", "AutoDjAddTop",
```

Comment:
In the library right-click menu we follow the ``add bottom`` and ``add top`` ordering.
In midi wizard it is ``Add top`` and ``Add bottom``